doc/docs/server-pull.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Pull Endpoint
+slug: /server-pull
+---
+
+The pull endpoint is used when doing a [`pull`](api/classes/replicache#pull).
+
+The purpose of the server pull endpoint is to send the client view to the
+client. The client view is the view of the data that the client should see.
```

Comment:
```suggestion
client.
```

The conceptual/overview doc should explain concepts. The reference should just be a reference. See also: https://diataxis.fr/